addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -303,6 +303,7 @@
                 <t t-set="title">Opportunities</t>
                 <t t-set="url" t-value="'/my/opportunities'"/>
                 <t t-set="placeholder_count" t-value="'opp_count'"/>
+                <t t-set="config_card" t-value="True if request.env.user.partner_id.grade_id or request.env.user.commercial_partner_id.grade_id else False"/>
```

Comment:
Hello @thsh-odoo , thank you for your work.

I think we can simplify as it will return a boolean expression:
`t-value="request.env.user.partner_id.grade_id or request.env.user.commercial_partner_id.grade_id"`

The Commit Message is quite clear, but maybe you can insist on the issue being on a portal feature:

-> change suggestion:  Enable users to create new opportunities **on the portal** even if they do not have any yet.

Thanks and cheers! :)